docs/contributing.rst

```diff
@@ -235,6 +235,8 @@ of the theme. The current font selections are:
 
   - allowing upgrades to be handled in a relatively sane, manageable way
 
+Currently, only fontawesome is vendored (other fonts like Lato are no longer
```

Comment:
Consider:
- normalizing case/branding to `FontAwesome 5 Free` and linking to it
- and/or mentioning its use in icon links (with crosslink to `user_guide/configuring.html`) and admonitions